### PR TITLE
Marketplace restructuring

### PR DESCRIPTION
**Pull request by Luyao-Zhang-1**

### CODE REVIEW

**Comment by Luyao-Zhang-1** [comment]
[PR7468](https://github.com/mendix/docs/pull/7468) is closed due to too many complicated merge conflicts. This branch is up to date with development. The work will continue here.

File: content/en/docs/appstore/creating-content/guidelines-content-creators.md [hunk 1]
```diff
@@ -0,0 +1,329 @@
...
+When you prepare a product and submit it to the public Marketplace, follow the guidelines in this document.
+
+## Creating Your Content
+
+For the guidelines for creating a specific type of content, see the specific documents:
+
+* Guidelines for Creating Widgets
+* Guidelines for Creating Modules
+  * Guidelines for Creating Solutions 
+* Guidelines for Creating Solutions
```
Comment (melvingrootenboers) [inline content/en/docs/appstore/creating-content/guidelines-content-creators.md]: Why is this shown twice? "Guidelines for Creating Solutions"

File: content/en/docs/appstore/creating-content/guidelines-content-creators.md [hunk 2]
```diff
@@ -0,0 +1,329 @@
...
+
+### General Guidelines{#general}
+
+Below are the general guidelines for creating new Marketplace content:
+
+* Set up a separate app to build and maintain your Marketplace component.
+* Use a relatively recent Mendix version when creating the item, not only the latest Mendix version.
```
Comment (melvingrootenboers) [inline content/en/docs/appstore/creating-content/guidelines-content-creators.md]: We should advice to use the Latests MTS or LTS of any major Studio Pro version. If that does not yet exist, than the earliest version of any major

File: content/en/docs/appstore/creating-content/guidelines-content-creators.md [hunk 3]
```diff
@@ -0,0 +1,329 @@
...
+* Ensure that any latest version of your product supports at least one Mendix LTS versions or Mendix MTS versions.
+* Create multiple versions of your Marketplace component, for example, for Studio Pro 8 and 9.
```
Comment (melvingrootenboers) [inline content/en/docs/appstore/creating-content/guidelines-content-creators.md]: Why? This is more about if you want your product to be adopted by projects in 8,9 and 10, or only 9, 10, or only 10. On the other hand, the more version you support, the more time you need to spend on keeping them all up to date

File: content/en/docs/appstore/creating-content/guidelines-content-creators.md [hunk 4]
```diff
@@ -0,0 +1,329 @@
...
+* Your products must be complete and fully functional, as advertised upon submission.
+
+When adding the content to the Marketplace, you can add information the end-user should know to the component [documentation](#doc).
```
Comment (melvingrootenboers) [inline content/en/docs/appstore/creating-content/guidelines-content-creators.md]: not "can", but "should"

File: content/en/docs/appstore/creating-content/guidelines-content-creators.md [hunk 5]
```diff
@@ -0,0 +1,329 @@
...
+4. If you add the *.mpk* file as a binary file to the release tag, the Mendix Marketplace automatically syncs the *.mpk* to your new draft:
+
+   {{< figure src="/attachments/appstore/sharing-content/github-releases.png" >}}
+
+5. Link this GitHub release to the upcoming Mendix Marketplace release by mentioning the GitHub release number in the description. For more details, see the [Package](#package) and [Updating Existing Marketplace Content](#updating) sections below.
+
+## Preparing Your Submission
+
+{{% alert color="info" %}}
+If you are not a component partner, you can skip this section.
```
Comment (melvingrootenboers) [inline content/en/docs/appstore/creating-content/guidelines-content-creators.md]: link to more information about a component partner. How do I know if I'm a component partner?

File: content/en/docs/appstore/creating-content/guidelines-content-creators.md (hunk 3 above)
Comment (Luyao-Zhang-1) [reply content/en/docs/appstore/creating-content/guidelines-content-creators.md]: This advice is currently in this [doc](https://docs.mendix.com/appstore/sharing-content/#general). Please see the third bullet. Shall I remove it?

File: content/en/docs/appstore/creating-content/guidelines-content-creators.md (hunk 5 above)
Comment (Luyao-Zhang-1) [reply content/en/docs/appstore/creating-content/guidelines-content-creators.md]: How about linking to [this doc](https://docs.mendix.com/appstore/partner-program/)?

File: content/en/docs/appstore/creating-content/guidelines-content-creators.md (hunk 1 above)
Comment (Luyao-Zhang-1) [reply content/en/docs/appstore/creating-content/guidelines-content-creators.md]: The first instance should be "Guidelines for Creating Connectors". I will fix it.

File: content/en/docs/appstore/creating-content/guidelines-content-creators.md [hunk 6]
```diff
@@ -0,0 +1,329 @@
...
+- Continuously monitor customer satisfaction with both the product itself and the support you are providing.
+- The support team should have a direct line of communication with the engineering team delivering the product as second- or third-line  support.
+- Put internal procedures in place to manage changes, incidents, and escalations.
+- Make a status page like status.mendix.com available for all customers.
+
+{{% alert color="info" %}}
+Mendix reserves the right to temporarily delist any products that do not adhere to these requirements until resolved. In principle, you will be notified in advance, but for critical issues, the notification can be sent at the same time as the delisting and customer notifications.
+{{% /alert %}}
+
+### Product Updates
```
Comment (Luyao-Zhang-1) [inline content/en/docs/appstore/creating-content/guidelines-content-creators.md]: @melvingrootenboers Could you provide input for this section?

File: content/en/docs/appstore/creating-content/guidelines-content-creators.md [hunk 7]
```diff
@@ -0,0 +1,329 @@
...
+
+{{% todo %}}No content here. Content to be added.{{% /todo %}}
+
+### Product Retiring & Removal
```
Comment (Luyao-Zhang-1) [inline content/en/docs/appstore/creating-content/guidelines-content-creators.md]: @melvingrootenboers Could you also provide input for this section?

**Pull request merged**